src/masoniteorm/connections/SQLiteConnection.py

```diff
@@ -34,10 +34,12 @@ def __init__(
 
     def make_connection(self):
         """This sets the connection on the connection class"""
-        print("db", self.database)
 
         self._connection = sqlite3.connect(self.database, isolation_level=None)
 
+        self._cursor = self._connection.cursor()
+        self._cursor.execute("PRAGMA foreign_keys = ON", ())
```

Comment:
This should only be done when modifying the schema. We shouldn't turn pragma keys on every time we make a connection here

Reply:
You're right. I'll move to sqlite grammar.